### PR TITLE
Sean Valdivia

### PR DESCRIPTION
**Pull request by NewbieWanKenobi**
This is where I'm at. I will keep improving it through the weekend.

### CODE REVIEW

File: Answers.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+1. If you saw this HTML: <div class="box box1 box2 box3"></div> which class has the most specificity weight?
+   The `box3` class would have the most specificity weight.
```
Comment (lefrenk) [inline Answers.md]: 👍

File: Answers.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+2. Describe the difference between display: block; and display: inline;.
+   `display: inline` places the element next to other elements on the same line as an inline element.
+   `display: block` places the element on a new line as a block element that takes up the whole width.
```
Comment (lefrenk) [inline Answers.md]: Awesome! 🙌

File: Answers.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+3. While using flexbox, what axis are you using when you use the property: align-items: center?
+  `align-items: center` centers items on the cross axis.
+
+4. What is the difference between fixed layout, adaptive layout, fluid layout, and responsive layout?
```
Comment (lefrenk) [inline Answers.md]: Excellent! 🤘

File: Answers.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+   * Fixed layout: The size of the website is fixed and doesn't change. Very rigid.
+   * Adaptive layout: The size of the website changes according to breakpoints. Better but still rigid.
+   * Fluid layout: The website tries to adjust to any viewport size no matter how small or big--
+     this can lead to some very strange results.
+   * Responsive layout: A truly responsive website is a combination is adaptive when it needs to be and
+     fluid when it needs to be; it is the best of both worlds.
+
+5. Why do we need to use the CSS property max-width on the outer-most container in a responsive website?
```
Comment (lefrenk) [inline Answers.md]: Nailed it.

File: index.html [hunk 5]
```diff
@@ -13,6 +13,22 @@
 
 <body>
     <div class="container">
+        <div class="navbar">
```
Comment (lefrenk) [inline index.html]: Instead of `<div>`, you can use `<header>` and `<nav>` to be more specific.

File: index.html [hunk 6]
```diff
@@ -13,6 +13,22 @@
...
+          <div class="nav-buttons">
+              <div class="logo-box">
+                  <img src="img/lambda-black.png" class="lambda-logo">
+              </div>
+              <div class="button-box">
+                  <p>Home</p>
```
Comment (lefrenk) [inline index.html]: Make sure they're `<a>` tags instead of `<p>`

File: index.html [hunk 7]
```diff
@@ -30,15 +46,15 @@ <h2>Why Did It Have To Be Boxes...</h2>
...
-                <div class="box">Box 10</div>
+                <div class="box box-2">Box 2</div>
+                <div class="box box-3">Box 3</div>
+                <div class="box box-4">Box 4</div>
+                <div class="box box-5">Box 5</div>
+                <div class="box box-6">Box 6</div>
+                <div class="box box-7">Box 7</div>
+                <div class="box box-8">Box 8</div>
+                <div class="box box-9">Box 9</div>
+                <div class="box box-10">Box 10</div>
```
Comment (lefrenk) [inline index.html]: 🙌

**Comment by lefrenk** [comment]
Hey Sean! You're off to a pretty good start here. Keep up the good work, you've got all of today and this weekend to finish 'er up.

**Comment by NewbieWanKenobi** [comment]
Will do I have tomorrow off. My boss is pretty understanding so that is a blessing. Plus he wants me to build a website for the restaurant I work at,  The Little Rhein Steakhouse.

**Comment by lefrenk** [comment]
Awesome! Sounds like a fun lil challenge

File: css/index.css [hunk 8]
```diff
@@ -166,6 +166,11 @@ footer nav a {
     align-items: flex-end;
 }
 
+.link {
```
Comment (lefrenk) [inline css/index.css]: Instead of assigning a class to the links, you could have used `.button-box a` as the selector

File: index.html [hunk 9]
```diff
@@ -3,7 +3,7 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (lefrenk) [inline index.html]: 👍

**Comment by lefrenk** [comment]
Awesome! Nice work with the 768px media query. If you want to do the 400px one, you're more than welcome.

Otherwise, you can close this pull request. Cheers! :)

**Pull request closed without merging**